module-2/backend/week_three.md

```diff
@@ -10,14 +10,32 @@ Note: When you're done, submit a PR with a reflection in the comments about how
 ### Questions
 
 1. What is the entry at the command line to create a new rails app?
+rails new <project name> <options>
 2. What do Models generally inherit from in rails?
+ ApplicationRecord
 3. What do Controllers generally inherit from in a rails project?
+ ActionController
 4. How would I create a route if I wanted to see a specific horse in my routes fitle assuming I'm sticking to standard conventions and that I didn't want other CRUD functionality?
+  Assuming there is not a route already.
+  resources :horses, only: :show
 5. What rake task is useful when looking at routes, and what information does it give you?
+  rake routes - it gives you information on all of the current routes.
 6. What is an example of a route helper? When would you use them?
+  A route helper is used to refer to a particular route.  I use these mostly in link helpers and redirects.
 7. What's the difference between what `_url` and `_path` return when combined with a routes prefix?
+  _path is the path relative to root and _url is an absolute path.
 8. What are strong params and why are the necessary?
+  Strong params define what parameters can be input for a particular parameter field for a controller.  This is neccesary to prevent invalid or unexpected paramters from being input through a controller.
 9. What role does `form_for` play in helping us create our forms?
+  It is a helper for generating a post or put form that can edit or create a record.
 10. How does `form_for` know where to submit the user's input?
+  It is based on the view it is rendered in.  If it is rendered in a 'new' view it will do a post, but if it is in an edit form it will do a PUT/PATCH.
```

Comment:
the instance variable passed provides the model the path will reference `form_for(@horse)` creates a post to the horses_path